core/ide/src/main/java/org/alice/stageide/sceneeditor/StorytellingSceneEditor.java

```diff
@@ -463,6 +475,44 @@ public void setSelectedExpression(Expression expression) {
     }
   }
 
+  private void updateOrthographicCameraView(EntityImp targetImplementation) {
+    double targetHeight = targetImplementation.getAxisAlignedMinimumBoundingBox().getHeight();
+    AffineMatrix4x4 targetTransform = targetImplementation.getAbsoluteTransformation();
+    ClippedZPlane picturePlane = new ClippedZPlane();
+
+    AffineMatrix4x4 topTransform = AffineMatrix4x4.createIdentity();
+    topTransform.translation.x = targetTransform.translation.x;
+    topTransform.translation.y = 10;
```

Comment:
Instead of a constant 10 this could be based on the model height to better handle very large or small models. Maybe 3x height?
If you wanted to get really fancy you could  make sure the model takes up some portion of the view, but that might get complicated.

Reply:
I make it 3x height. Yeah, that is a good suggestion! We can add the portion one in the future haha